datasets.md

```diff
@@ -79,6 +79,10 @@ In an effort to continue diversifying the data being onboarded onto the network,
 | CLEVR Dataset | A Diagnostic Dataset for Compositional Language and Elementary Visual Reasoning. | 19 GB | png&json | https://www.kaggle.com/timoboz/clevr-dataset |
 | COCO2017 | Common Objects in Context | 20 GB | jpg | https://www.kaggle.com/aishwr/coco2017 |
 | COCO 2017 Dataset | COCO is a large-scale object detection, segmentation, and captioning dataset. | 26 GB | jpg | https://www.kaggle.com/awsaf49/coco-2017-dataset |
+| PandaSet Dataset | Public large-scale dataset for autonomous driving |  31GB | Various | https://www.kaggle.com/usharengaraju/pandaset-dataset |
+| AudioSet [Train] | Training audio files for AudioSet |  29GB | Various | https://www.kaggle.com/zfturbo/audioset |
```

Comment:
A little lower leverage dataset and is not public domain for sharing (https://creativecommons.org/licenses/by-sa/4.0/). 673 views, 90 downloads. Though this could be useful, given the licensing requirements, might be better to replace this with something similar that is creative commons public domain.

Reply:
AudioSet use (https://creativecommons.org/licenses/by-sa/4.0/)   same as https://research.google.com/audioset/download.html     I will remove it.     
I hope we can  use CC-BY-SA datasets in slingshot in the future.     (SA seems is easy to follow)
And we can store metadata(such as title, license) in filecoin in the future.